tests/network/test_services.py

```diff
@@ -74,13 +80,21 @@ def test_get_transactions_main_failure(self):
         with pytest.raises(ConnectionError):
             MockBackend.get_transactions(MAIN_ADDRESS_USED1)
 
-    def test_get_transactions_test_equal(self):
-        results = [call(TEST_ADDRESS_USED2)[:100] for call in NetworkAPI.GET_TRANSACTIONS_TEST]
-        assert all_items_common(results)
+    def test_get_transaction_by_id_main_equal(self):
+        results = [calc_txid(call(MAIN_TX_VALID)) for call in NetworkAPI.GET_TRANSACTION_BY_ID_MAIN]
+        assert all_items_equal(results)
 
-    def test_get_transactions_test_failure(self):
+    def test_get_transaction_by_id_main_failure(self):
         with pytest.raises(ConnectionError):
-            MockBackend.get_transactions_testnet(TEST_ADDRESS_USED2)
+            MockBackend.get_transactions(MAIN_TX_VALID)
```

Comment:
We should call `get_transaction_by_id` on the `MockBackend`.

Reply:
Done. This was also a mistake (see above).